app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    def __repr__(self) -> str:
+        return f"Person(name={self.name}, age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+
+    for data in people:
+        person = Person(data["name"], data["age"])
+        persons.append(person)
```

Comment:
Prawidłowe,  chociaż można tutaj fajnie wykorzystać list comprehension:
```suggestion
    persons = [Person(person["name"], person["age"]) for person in people]
```